homeassistant/components/scrape/config_flow.py

```diff
@@ -102,19 +102,22 @@
     ),
     vol.Optional(CONF_ATTRIBUTE): TextSelector(),
     vol.Optional(CONF_VALUE_TEMPLATE): TemplateSelector(),
-    vol.Optional(CONF_DEVICE_CLASS): SelectSelector(
+    # Note: we set default to ensure that frontend does not omit the result
+    vol.Optional(CONF_DEVICE_CLASS, default=""): SelectSelector(
         SelectSelectorConfig(
-            options=[cls.value for cls in SensorDeviceClass],
+            options=[""] + [cls.value for cls in SensorDeviceClass],
```

Comment:
Instead of this, use a sentinel value which is less error prone than the empty string